### ISSUE TITLE
[Accessibility] Add accessibility checks to Travis

### ISSUE THREAD

**Issue by KyleJu** [web-platform-tests/wpt.fyi#2473]
The accessibility check is removed from Travis due to https://github.com/web-platform-tests/wpt.fyi/issues/2450. It is fine because we won't ship major UI changes in the near term. In the long term, we should investigate the lighthouse errors and put the accessibility check back to prevent regressions.

**Comment by KyleJu** [comment]
Note that the I'm unable to generate lighthouse accessibility report through devtools for wpt.fyi, which was working a few days when fixing #2450.

**Comment by KyleJu** [comment]
https://github.com/GoogleChrome/lighthouse/issues/6512#issuecomment-797152215 could be a fix for the #2450, although the cause of `NO_FCP` is still unknown.
